src/tests/slackbot_test.py

```diff
@@ -16,3 +16,9 @@ def test_fail_send_message_to_slack():
     is_success = bot.send_slander_message_to_slack("test")
     assert is_success == False
 
+def test_send_message_to_slander_api(self):
+    bot = Slackbot()
+    is_success = bot.send_message_to_slander_api("お前馬鹿かよ")
+    self.assertEqual(is_success, [0.9252830743789673])
```

Comment:
You can't use self in this function because this function doesn't belong to class.
So maybe you fix
`def test_send_message_to_slander_api():`
and
`assert is_success == [0.9252830743789673]`

Reply:
Thanks! I edited it!